pkg/dependency/parser/java/pom/parse.go

```diff
@@ -148,7 +148,8 @@ func (p *Parser) parseRoot(root artifact, uniqModules map[string]struct{}) ([]ft
 			if _, ok := uniqModules[art.String()]; ok {
 				continue
 			}
-			uniqModules[art.String()] = struct{}{}
+			art.ID = uuid.New()
```

Comment:
I want the output reports to be as reproducible as possible. I'm considering another approach, but I have not yet come up with...

Reply:
Do you mean using something like a hash instead of UUID?

Reply:
That's one of the options. Regardless of the method, I want the same value to be output as much as possible when scanning the same object.